frontend/src/components/object/ObjectUpload.vue

```diff
@@ -1,15 +1,12 @@
 <script setup lang="ts">
-// Vue
 import { ref } from 'vue';
 import { useRoute } from 'vue-router';
-// PrimeVue
 import { useToast } from 'primevue/usetoast';
 import Button from 'primevue/button';
 import FileUpload from 'primevue/fileupload';
-// State
-import { useObjectStore } from '@/store';
-// Component
+
```

Comment:
Style: We should try to reorder the primevue/* imports to come before the vue ones now that we're removing the comment groupings on them unless that causes a compilation behavior due to import ordering.